tests/lms/util/lti_test.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from pyramid.httpexceptions import HTTPBadRequest
+
+from lms.util import lti_params_for
+
+
+class TestLTIParamsFor:
+    def test_with_lti_launch_request(self, pyramid_request):
```

Comment:
These test names differ from our typical test-naming conventions. I...think they're fine, really, but wanted to mention it.